index.src.html

```diff
@@ -130,8 +130,16 @@ <h3 id="examples">Example</h3>
   </pre>
 
   Before blocking this insecure request as mixed content, the user agent
-  performs an HSTS priming request to the root of the resource's host, as
-  follows:
+  consults its <a lt="matches the HSTS cache">HSTS cache</a> for the origin of
+  the resource.
+
+  If the origin of the resource matched the HSTS cache, the user agent can then
```

Comment:
Step 5 of http://mikewest.github.io/hsts-priming/#modifications-fetch-priming. Basically "Do whatever." :)

Maybe ~24 hours is reasonable, maybe we should look at the response's cache headers, maybe maybe maybe. *shrug* Suggestions welcome. :)